recipes/openmm-xtb/build.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+rm -rf build || true
+
+
+CMAKE_FLAGS="${CMAKE_ARGS} -DCMAKE_INSTALL_PREFIX=${PREFIX} -DCMAKE_BUILD_TYPE=Release"
+CMAKE_FLAGS+=" -DPython3_EXECUTABLE=${PYTHON}"
+CMAKE_FLAGS+=" -DOPENMM_DIR=${PREFIX}"
+
+
+mkdir build && cd build
+cmake ${CMAKE_FLAGS} ${SRC_DIR}
+make -j$CPU_COUNT
+make -j$CPU_COUNT install PythonInstall
+
+# Copy tests to the share folder
+# Include test executables too
+TEST_DIR="${PREFIX}/share/${PKG_NAME}/tests/"
+
+mkdir -p "${TEST_DIR}"
+cp -a ../python/tests/* tests/Test* "${TEST_DIR}"
+if [[ "$target_platform" == osx* ]]; then
+    find "${TEST_DIR}" -name "Test*" -perm +0111 -type f \
+        -exec python $RECIPE_DIR/patch_osx_tests.py "{}" \; \
```

Comment:
Can we check if we still need this?

Reply:
It's still needed.  WIthout it we get the `conda_build.exceptions.OverLinkingError: overlinking check failed` error.

Reply:
For future reference, this is the full error:

```
  ERROR (openmm-xtb,share/openmm-xtb/tests/TestXtbForce): /Users/runner/Miniforge3/conda-bld/openmm-xtb_1712618910781/work/build/libOpenMMXTB.dylib not found in packages, sysroot(s) nor the missing_dso_whitelist.
```

Where we can see that the `build_dir` path is hardcoded so it doesn't end up being replaced by the PREFIX machinery in conda-build. To me this sounds like something that could be fixed in the CMake config, preferrably.

That said I don't know exactly where that'd go so I'd support this workaround for now.

@xhochy, what do you think?

Reply:
Sounds like a good workaround. We have tried to fix  this in (upstream) Arrow, too, but we never found a reliable solution :(

Reply:
Thanks!  Is it ok to merge this?

Reply:
From my PoV yes. I'll let @jaimergp have a final look.